src/stirling/obj_tools/elf_reader.cc

```diff
@@ -519,12 +570,26 @@ std::vector<uint64_t> FindRetInsts(utils::u8string_view byte_code) {
   return res;
 }
 
+StatusOr<Arch> GetArchFromELFMachine(ELFIO::Elf_Half machine) {
+  switch (machine) {
+    case ELFIO::EM_X86_64:
+      return ArchX86_64;
+    // ELFIO doesn't have the updated versions of EM_*, so we use the aarch64 value directly.
+    case 183:
```

Comment:
should we either update our fork with main or just update this one line in our fork from main?
https://github.com/serge1/ELFIO/blob/main/elfio/elf_types.hpp#L250